solver.go

```diff
@@ -39,7 +39,7 @@ type Solver interface {
 	// domain socket in the build container.
 	Forward(src, dest string, opts ...ForwardOption) llb.RunOption
 	// Local will add a local directory to the solve request and session.
-	Local(name string, opts ...llb.LocalOption) llb.State
+	Local(name, path string, opts ...llb.LocalOption) llb.State
```

Comment:
Why this change?  it doesnt seem necessary.  I was trying to keep it consistent with the llb.Local api.

Reply:
The new `dockerfile2llb` code looks for a local named `context`, and it's hard to override that behavior.

Reply:
Hmm, maybe we can just create an llb.LocalOption that allows us to customize the name in the case for dockerfile2llb?  Then the 1st required argument is just `path`?